### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by ebiwonjumit**
Made a Error check change, to permit 0 as a value

### CODE REVIEW

File: extensions/BeatsBlox/index.js [hunk 1]
```diff
@@ -5888,8 +5888,8 @@
                         return new List(pattern.map((x) => rootNote + x));
                     }),
                     new Extension.Block('setTrackEffect', 'command', 'music','track %supportedEffects effect to %n %', ['Volume','50'], function (effectName, level){
-                        if(parseInt(level) > 100 || level == ''){
-                            throw Error('Level must be a value between 1 and 100');
+                        if(parseInt(level) > 100 || parseInt(level) < 0 || level == '' ||){
```
Comment (dragazo) [inline extensions/BeatsBlox/index.js:5891]: the trailing `||` in this expression is a syntax error and would break the extension

Comment (ebiwonjumit) [reply extensions/BeatsBlox/index.js]: Removed it. Thanks for catching that!

**Comment by dragazo** [comment]
I thought you mentioned that the lower limit was `1` because entering `0` for some effects gives an exception. Is that no longer the case?

**Comment by ebiwonjumit** [comment]
Yes, I added checks for those effects that would be affected by a `0` value.

**Comment by dragazo** [comment]
Where? The only change in this PR is bounds checking the input from the user. Actually allowing zero would have to be in Will's library unless you do some other trick in this one.

**Comment by ebiwonjumit** [comment]
The Reverb and Echo inputs are bounded below, which I had added in previously:
`          if(effectName == "Echo" && level > 95){
                            throw Error("Echo: value cannot be greater than 95")
                        }
                        if(effectName == "Reverb" && level < 10){
                            throw Error("Reverb: value cannot be less than 10")
                        }`

**Comment by ebiwonjumit** [comment]
Im gonna git diff all your changes today anyway, so we can close this PR. Ill add it in through my feature branch and we can review it with all the code thats been added.

**Comment by dragazo** [comment]
Sure, also while you're applying changes, it would probably be good to just load in Will's library dynamically (like [this](https://github.com/NetsBlox/extensions/blob/8bb6310837326d0e6f907c7b77c1f98a54a775c0/extensions/TuneScope/index.js#L553-L557)) instead of using rollup. That'll make it a lot less confusing which parts are actually the extension vs Will's API. If he doesn't host a version that works for that, you can ask him about it. Even just declaring a global handle or something in the `window` object would be fine.

**Comment by dragazo** [comment]
Also, if you load in Will's library dynamically, you can just take the new version of the extension from `master`, delete the part that's Will's library bundled in from rollup, add the dynamic import, and you'd be done (no need to cherry-pick my changes).

**Pull request closed without merging**